src/DeFuncto.Tests/Core/Types/AsyncOption/Bind.cs

```diff
@@ -20,19 +20,25 @@ public class Bind
         Some(a)
             .Async()
             .Bind(_ => None.Option<int>().ToTask())
-            .ShouldBeNone();
+            .ShouldBeNone()
+            .GetAwaiter()
+            .GetResult();
 
     [Fact(DisplayName = "Skips none after none")]
     public void NoneOnNone() =>
         None.Option<string>()
             .Async()
             .Bind(_ => None.Option<int>())
-            .ShouldBeNone();
```

Comment:
Why not making a seperate test for the GetAwaiter cases?

Reply:
There is a separate test for the getawaiter cases. This change is not related to the getawaiter.

Reply:
This is getting the awaiter for `Task<Unit>` which is part of the BCL.